pom.xml

```diff
@@ -325,6 +325,25 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.mortbay.jetty</groupId>
+            <artifactId>jetty-util</artifactId>
+            <version>6.1.26</version>
+        </dependency>
+
+        <dependency>
```

Comment:
Move this above the "discard all logging from hadoop" comment (line 298) so that it's grouped with the other Hadoop dependencies.